Model/ManagerScore.cs

```diff
@@ -0,0 +1,46 @@
+﻿using static Model.ValidationTest;
+
+namespace Model
+{
+    public static class ManagerScore
+    {
+        public static List<double> MaxPerformanceScore { get; private set; }
+
+        /// <summary>
+        /// Calculates the managers performance score
+        /// </summary>
+        /// <param name="manager"></param>
+        /// <returns></returns>
+        public static double GetPerformanceScore(Manager manager)
+        {
+            double score = (double)(manager.RowsRead + manager.RowsWritten) / (manager.Runtime.HasValue ? manager.Runtime.Value.Seconds : 0);
```

Comment:
no no no, hvis en manager runtime ikke har value, dividerer du med 0 som er no go! det crasher programmet, så måske skal du gøre sådan her:
```suggestion
            double score = 0;
            if (manager.Runtime.HasValue) {
                score = (double)(manager.RowsRead + manager.RowsWritten) / manager.Runtime.Value.Seconds;
```